db/migrate/20150710211530_change_events_exception_to_text.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeEventsExceptionToText < ActiveRecord::Migration
+  def up
+    change_column :events, :exception, :text
```

Comment:
@coblej Good catch! I didn't even look at the schema and had forgotten about that.